packages/graphics/graphics.5.1.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "The OCaml graphics library"
+description: """
+The graphics library provides a set of portable drawing
+primitives. Drawing takes place in a separate window that is created
+when Graphics.open_graph is called.
+
+This library used to be distributed with OCaml up to OCaml 4.08.
+"""
+maintainer: ["jeremie@dimino.org" "david.allsopp@metastack.com"]
+authors: [
+  "Xavier Leroy" "Jun Furuse" "J-M Geffroy" "Jacob Navia" "Pierre Weis"
+]
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "https://github.com/ocaml/graphics"
+doc: "https://ocaml.github.io/graphics/"
+bug-reports: "https://github.com/ocaml/graphics/issues"
+depends: [
+  "dune" {>= "2.0"}
```

Comment:
```suggestion
  "dune" {>= "1.11"}
  "dune-configurator"
```
Sorry but this was not a proper fix (see exact same error than before in the CI)

Reply:
Indeed. I added the dependency on `dune-configurator`. However, we also need dune >= 2.0 as the code itself references `dune-configurator` rather than `dune.configurator`. The former is not available with dune 1.x.